### DISCUSSION TITLE
Templated Faceplate / Views

### DISCUSSION THREAD

**Discussion by henjoe** [frangoteam/FUXA#761] [General]
Hi I created a basic faceplate template on a specific view. What I wanna do is I want this to be the faceplate for all the tags that can be seen to another view. 

Let say for example when I press button in View1, it will open my faceplate, and the data on my faceplate will be the data related to the tag specified in my view1. In other SCADA I can do this via binding, I just don't know how am I going to do this in Fuxa.

![image](https://user-images.githubusercontent.com/49782655/227759144-ff86482c-7541-4ed4-9768-8e6d1ba60ba3.png)

This is very helpful because instead of creating multiple view for each tags, you can simply define a dynamic template view that can be reuse by other view.

Hope someone can help me.

Thanks and Regards,
Henjoe

**Comment by unocelli** [comment]
Hi,
you can do this, but you can't take opened the same card at the same time [wiki](https://github.com/frangoteam/FUXA/wiki/HowTo-use-same-view)

**Reply by henjoe** [reply]
Thanks @unocelli ! Works well. And I hope in the future it can be open in the same time. 

Regards,
Henjoe
